index.html

```diff
@@ -80,7 +81,7 @@ <h1 class="font">
         </div>
       </section>
       <div class="works" id="portfolio">
```

Comment:
[ ] Currently, your design is not matched with the Figma design. Kindly reduce the gap between the main page and the pop-up and try to match with the Figma design. :+1:

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![y2](https://user-images.githubusercontent.com/70483835/168184431-92c59b8b-cb8b-4c1b-a23e-b047350da50e.png) | ![f2](https://user-images.githubusercontent.com/70483835/168184425-3a4e8360-5f43-4e4b-8639-c5cf75ee23bb.png)


- [ ] Kindly do the adjustment for mobile also. :+1:

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![y3](https://user-images.githubusercontent.com/70483835/168185038-d059a776-4cc0-4a1f-b393-54c848f816a6.png) | ![f3](https://user-images.githubusercontent.com/70483835/168185032-5b3f952a-78c0-4238-9929-e1c31e61040f.png)

- [ ] Please try to match the font color according to the Figma design. :+1:

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![y1](https://user-images.githubusercontent.com/70483835/168185212-0b0bc1a3-689c-450d-b8ea-0a3dc0cd3ec9.png) | ![f1](https://user-images.githubusercontent.com/70483835/168185210-3cfed4cc-4d57-4dac-9299-6b5535226550.png)